tests/installation/user_settings.pm

```diff
@@ -35,23 +35,22 @@ sub run {
         return;
     }
 
-    if (check_var('SLE_PRODUCT', 'hpc')) {
-        # in case of HPC product we okay to ignore mistyped username
-        $self->enter_userinfo(max_interval => utils::VERY_SLOW_TYPING_SPEED);
-        assert_screen([qw(inst-userinfostyped-ignore-full-name inst-userinfostyped-expected-typefaces)]);
-    }
-    else {
+    if (get_var('ASSERT_BSC1122804')) {
```

Comment:
+1 , I think it is best option which would make everyone happy

Reply:
Could you please update PR's description? You have old name there.